test/Boltzmann_Tst.Jl

```diff
@@ -0,0 +1,78 @@
+using HallPIC: HallPIC as hp 
+using Plots
+using Statistics
+
+const Xenon = hp.Gas(name=:Xe, mass=131.293)
+
+
+T = Float64
+# general initialization 
+n_iterations = 1000
+n_cell = 10
+n_n = 5000
+dt = T(75e-9 / hp.t_0)
+grid = hp.Grid(n_cell, 0, 0.1 / hp.x_0, 0.1 / hp.x_0^2, hp.OpenBoundary(), hp.OpenBoundary())
+E = zeros(T, length(grid.face_centers))
+phi = zeros(T, length(grid.cell_centers))
+
+# initialize neutrals 
+neutral_properties = hp.SpeciesProperties{T}(n_cell+2, Xenon(0))
+neutral_properties.dens .= 1e19 / hp.n_0# 1/m^3
+neutral_properties.vel .= 0.0
+neutral_properties.temp .= (500 / 11604) # eV
+neutral_properties.avg_weight .= 0
+neutral_properties.N_particles .= 0
+neutrals = hp.initialize_particles(neutral_properties, grid, n_n)
+# deposit to grid 
+hp.locate_particles!(neutrals, grid)
+hp.deposit!(neutral_properties, neutrals, grid)
+
+
+# initialize ions 
+ion_properties = hp.SpeciesProperties{T}(n_cell+2, Xenon(1))
+ion_properties.dens .=  1e16 / hp.n_0# 1/m^3
+ion_properties.vel .= 0.0
+ion_properties.temp .=  0.1 # eV
+ion_properties.avg_weight .= 0
+ion_properties.N_particles .= 0
+
+ions = hp.initialize_particles(ion_properties, grid, n_n)
+
+# deposit to grid 
+hp.locate_particles!(ions, grid)
+hp.deposit!(ion_properties, ions, grid) 
+
+
+# initialize some electron properties
+electron = hp.Gas(name=:e, mass=0.00054858)
```

Comment:
factor out magic numbers into constants, 0.00054858 seems to come out of nowhere. something like const m_e = 0.0054858 in the main package and a comment that says this is the normalized electron mass